app/workers/package_manager_download_worker.rb

```diff
@@ -39,12 +39,7 @@ def initialize(platform, name, version)
     inqlude: PackageManager::Inqlude,
     julia: PackageManager::Julia,
     maven: PackageManager::Maven,
-    maven_atlassian: PackageManager::Maven::Atlassian,
-    maven_hortonworks: PackageManager::Maven::Hortonworks,
     maven_mavencentral: PackageManager::Maven::MavenCentral,
-    maven_springlibs: PackageManager::Maven::SpringLibs,
-    maven_jboss: PackageManager::Maven::Jboss,
```

Comment:
was there a reason for stopping JBoss? I still see about 3k updates from depper in the past month for `maven_jboss`.